architecture/core_concepts/routes.adoc

```diff
@@ -621,6 +622,11 @@ the destination. This field is required, but only for re-encryption.
 [[router-sharding]]
 == Router Sharding
 
+An OpenShift administrator can deploy multiple routers in an OpenShift cluster. Since the
+routers always listen on ports 80 (http) and 443 (https), the routers are placed on
+nodes where the ports are available. Also, the service account for the routers must
```

Comment:
I don't like the also...  how about:

In order to have permission to read the namespace labels, the service account a sharded router is running with must have the `cluster-reader` permission.